docs/client-sdk/javascript/tutorials/label-conversations.md

```diff
@@ -5,13 +5,13 @@ sidebar_position: 4
 
 # Use labels to differentiate multiple conversations between a pair of addresses
 
-With XMTP, a pair of blockchain account addresses can have multiple ongoing conversations. In addition, with XMTP, an app can display all conversations a user has, regardless of the app they used to create the conversation. This concept is known as a portable, or universal, inbox.
+With XMTP, a pair of blockchain account addresses can have multiple ongoing conversations. In addition, with XMTP, an app can display all conversations a user has, regardless of the app they used to create the conversation. This concept is known as an interoperable inbox.
 
 Alternatively, an app can implement conversation IDs and metadata to [filter conversations](filter-conversations). For example, a developer can choose to assign app-specific conversation IDs and metadata to filter and display only conversations created by their app.
```

Comment:
A nugget that Nick shared in Discord that would be great to add as a tip paired with an example: Our recommendation is to only show one conversation per address & conversationID combination in your UI so that users will be talking on the same topic.

Reply:
Fantastic - added a callout.